scripts/ingests/zjzhang/zj_modeled_parameters_no_source.csv

```diff
@@ -0,0 +1,68 @@
+# %ECSV 1.0
+# ---
+# datatype:
+# - {name: Skipped, datatype: string}
+# - {name: Reason, datatype: string}
+# delimiter: ','
+# schema: astropy-2.0
+Skipped,Reason
+Calar 21,"no source found, L6T6"
+Calar 21,"no source found, L6T6"
+Calar 21,"no source found, L6T6"
+Calar 21,"no source found, L6T6"
+Calar 22,"no source found, L6T6"
+Calar 22,"no source found, L6T6"
+Calar 22,"no source found, L6T6"
+Calar 22,"no source found, L6T6"
+51 Eri b (cold-start),"no source found, L6T6"
+51 Eri b (cold-start),"no source found, L6T6"
+51 Eri b (cold-start),"no source found, L6T6"
+51 Eri b (cold-start),"no source found, L6T6"
+SDSS J105213.51+442255.7A,"no source found, L6T6"
+SDSS J105213.51+442255.7A,"no source found, L6T6"
+SDSS J105213.51+442255.7A,"no source found, L6T6"
+SDSS J105213.51+442255.7A,"no source found, L6T6"
+2MASSW J1728114+394859B,"no source found, L6T6"
```

Comment:
2MASS J17281150+3948593 is in the database, but no companions.  Maybe open a new issue or add the companion and companion relationship.